tests/test_reco_binary.py

```diff
@@ -66,6 +66,92 @@ def test_ctr_on_secondary_col(self):
         self.assertEqual(1. / 2, results['ctr'])
         self.assertEqual(2, results['support'])
 
+    def test_ips_ctr(self):
+        # Test immediate calculation of CTR
+        metric = BinaryRecoMetrics.CTR(click_column='click', estimation='ips')
+        actual = pd.DataFrame({Constants.user_id: [1, 2, 3, 4],
+                               Constants.item_id: [1, 2, 0, 3],
+                               'click': [0, 1, 0, 0]})
```

Comment:
curiosity question: if I use the IPS with no click-probability as in here, which means it will treat them uniformly, do I get back the "same" result as running CTR with Direct method?